src/rbac/defaultRoleManager.ts

```diff
@@ -123,6 +123,8 @@ export class DefaultRoleManager implements RoleManager {
   private maxHierarchyLevel: number;
   private hasPattern = false;
   private hasDomainPattern = false;
+  private hasDomainHierarchy = false;
+  private domainHierarchyManager: RoleManager;
```

Comment:
Or `domainHierarchyMngr` to follow `matchingFunc` and `domainMatchingFunc` naming schema?